edelphi/src/main/webapp/_scripts/gui/panel/queryblockcontroller.js

```diff
@@ -17,34 +17,23 @@ var QueryBlockScaleGraphFragmentController;
 var QueryBlockMultiSelectFragmentController;
 var QueryCommentsController;
 
-var QueryBlockController = Class
-    .create(
-        BlockController,
-        {
-          initialize : function($super) {
-            $super();
-
-            this._nextButtonClickListener = this._onNextButtonClick
-                .bindAsEventListener(this);
-            this._finishButtonClickListener = this._onFinishButtonClick
-                .bindAsEventListener(this);
-            this._previousButtonClickListener = this._onPreviousButtonClick
-                .bindAsEventListener(this);
-            this._skipButtonClickListener = this._onSkipButtonClick
-                .bindAsEventListener(this);
-            this._skipLastButtonClickListener = this._onSkipLastButtonClick
-                .bindAsEventListener(this);
-            this._toggleCommentsClickListener = this._onToggleCommentsClickListener
-                .bindAsEventListener(this);
-            this._sortCommentsAscendingTimeClickListener = this._onSortCommentsAscendingTimeClickListener
-                .bindAsEventListener(this);
-            this._sortCommentsDescendingTimeClickListener = this._onSortCommentsDescendingTimeClickListener
-                .bindAsEventListener(this);
-
-            this._currentPage = 0;
-            this._nextPageNumber = null;
-            this._previousPageNumber = null;
-          },
+var QueryBlockController = Class.create(BlockController, {
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png 'Severity: BLOCKER') "BlockController" does not exist. Change its name or declare it so that its usage doesn't result in a "ReferenceError". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=javascript%3AS3827)